unison-cli/src/Unison/Codebase/Editor/HandleCommand.hs

```diff
@@ -199,11 +199,10 @@ commandLine config awaitInput setBranchRef rt notifyUser notifyNumbered loadSour
     ClearWatchCache -> lift $ Codebase.clearWatches codebase
     FuzzySelect opts display choices -> liftIO $ Fuzzy.fuzzySelect opts display choices
 
-  watchCache (Reference.DerivedId h) = do
-    m1 <- Codebase.getWatch codebase WK.RegularWatch h
-    m2 <- maybe (Codebase.getWatch codebase WK.TestWatch h) (pure . Just) m1
-    pure $ Term.amap (const ()) <$> m2
-  watchCache Reference.Builtin{} = pure Nothing
+  watchCache :: Reference.Id -> IO (Maybe (Term v ()))
```

Comment:
`Codebase.lookupWatchCache`'s implementation was duplicated inside the definition of `watchCache` over here. Now it's not.